src/main/scala/no/ndla/searchapi/service/search/SearchConverterService.scala

```diff
@@ -889,13 +891,26 @@ trait SearchConverterService {
                 val contextFilters =
                   getFilters(resource, subject, bundle, bundle.resourceFilterConnections, filterVisibles)
                 val pathIds = (resource.id +: topicPath :+ subject.id).reverse
+                val relevanceId =
+                  subjectConnections
+                    .find(sc => sc.subjectid == subject.id)
+                    .get
+                    .relevanceId
+                    .getOrElse("urn:relevance:core")
```

Comment:
Hadde det vært bedre om vi ikke kaster en exception her om vi ikke finner subjectId'en?

```suggestion
                  subjectConnections
                    .find(sc => sc.subjectid == subject.id)
                    .flatMap(_.relevanceId)
                    .getOrElse("urn:relevance:core")
```

Reply:
Enig. Endrer.